docs/concepts.md

```diff
@@ -52,18 +52,18 @@ third-party validators (e.g. HL7 FHIR Validator).
 
 ## Reference Implementations
 
-An Inferno Test Kit may provide one or more reference implementations, which
-is a program that implements all requirements from a specification (e.g. FHIR) 
+Reference implementations are example servers that implement all requirements from a specification (e.g. FHIR) 
 and demonstrates "correct" behavior. They can
 be useful to develop tests against or to help interact with third-party
 solutions. For example, Inferno has a [Reference Server](https://github.com/inferno-framework/inferno-reference-server)
 for US Core and the Inferno ONC (g)(10) certification tests.
 
 ## Inferno Deployment
 
-A web host running one or more Inferno Test Kits. An example is the 
+An Inferno Deployment is a web host running one or more Inferno Test Kits. An example is the 
 [https://inferno.healthit.gov](https://inferno.healthit.gov) website.
 An individual Test Kit can also be run as an Inferno Deployment on 
-users' local machines without any additional
-configuration. An Inferno Deployment includes a web interface as well as
```

Comment:
Sometime fairly soon we'll want to introduce the concept of an 'Inferno Test Kit Platform' ([draft template here)](https://github.com/inferno-framework/inferno-platform-template).  Not quite yet though, but just reminding myself that we should update these once that is ready.